Formula/llvm.rb

```diff
@@ -303,3 +389,101 @@ def caveats
     end
   end
 end
+
+__END__
```

Comment:
This patch is a bit big. Can we put this in Homebrew/formula-patches?

Reply:
Sure, I'll move it there.